src/main/resources/fileTemplates/testMeIncludes/TestMe macros.groovy.ft

```diff
@@ -71,7 +71,7 @@
 ##
 #macro(grRenderMockedFields $testedClassFields)
 #foreach($field in $testedClassFields)
-#if($MockitoMockBuilder.isMockable($field))
+#if($hasMocks && $MockitoMockBuilder.isMockable($field))
```

Comment:
_$hasMocks_ variable is not defined in scope of this macro file or function. It works since Velocity puts it in context from the template that includes this macro. But it's a bad practice since the dependency on _$hasMocks_ is not explicitly expressed. This can easily become a bug if this variable would be removed from calling template. when changing code in calling template - the dependency on _hasMocks_ is transparent.
Options to resolve this:
1. Check _$hasMocks_ in a condition before calling _grRenderMockedFields_ (in such beware of added new lines and indentation)
2. pass _$hasMocks_ as an argument to  _grRenderMockedFields_ from the calling template (as in _grRenderTestSubjectInit_ function defined above)